code/extensions/FluentCMSPageEditController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+class FluentCMSPageEditController extends DataExtension {
+
+	public function doResetTranslations($data, $form) {
+		$record = $this->owner->getRecord($data['ID']);
+		$record->resetTranslations();
+
+        $this->owner->getResponse()->addHeader("X-Pjax","Content");
```

Comment:
Indentation here is a bit broken. :D Can you also please add phpdoc on the method?:D